org-roam-utils.el

```diff
@@ -45,7 +45,7 @@ Like file-name-extension, but does not strip version number."
   (save-match-data
     (let ((file (file-name-nondirectory filename)))
       (if (and (string-match "\\.[^.]*\\'" file)
-	             (not (eq 0 (match-beginning 0))))
+                     (not (eq 0 (match-beginning 0))))
```

Comment:
This was just changing a tab to a space

Reply:
Thanks! I copied this from Emacs, I guess they use tabs :-1: